src/Pagerfanta/Pagerfanta.php

```diff
@@ -134,6 +139,11 @@ public function setMaxPerPage($maxPerPage)
         return $this;
     }
 
+    /**
+     * @param $maxPerPage
```

Comment:
all your phpdoc is invalid btw. It is missing the type between ``@param`` and the name